lib/web/css/source/lib/_resets.less

```diff
@@ -49,7 +49,6 @@
 
     img {
         max-width: 100%;
-        height: auto;
         border: 0;
     }
```

Comment:
Remove height here may cause image distorted

Reply:
So why is there methods 'logo_height' inside the block if we can't use that?

Reply:
i'm ok with height auto in file reset. Only need remove redundant code in folder theme when you already have set default in reset file

Reply:
I would use both define width height attribute image html and also use height auto in css